src/main/resources/db/changelog/version-1.0/20.07.2023.create-payment-table.sql

```diff
@@ -1,9 +1,9 @@
 CREATE TABLE IF NOT EXISTS payment
 (
     payment_id        BIGSERIAL PRIMARY KEY,
-    payment_intent_id VARCHAR(32) NOT NULL UNIQUE,
+    payment_intent_id VARCHAR(64) NOT NULL UNIQUE,
+    shopping_session_id UUID NOT NULL UNIQUE,
     items_total_price DECIMAL    NOT NULL CHECK (items_total_price > 0),
     status            VARCHAR(32),
     description       TEXT
-
 );
```

Comment:
i guess we have to define cons on table lvl

Reply:
Done